frontend/src/app/projects/project-overview/project-overview.component.html

```diff
@@ -70,15 +70,15 @@
         >
           <div class="header flex-row items-center gap-2">
             @if (project.type === "training") {
-              <mat-icon>school</mat-icon>
+              <mat-icon class="shrink-0">school</mat-icon>
             }
             @if (project.visibility === "private") {
-              <mat-icon>lock</mat-icon>
+              <mat-icon class="shrink-0">lock</mat-icon>
             } @else {
-              <mat-icon>lock_open</mat-icon>
+              <mat-icon class="shrink-0">lock_open</mat-icon>
             }
 
-            <span class="">{{ project.name }}</span>
+            <span class="line-clamp-2">{{ project.name }}</span>
```

Comment:
```suggestion
            <span class="line-clamp-1">{{ project.name }}</span>
```

We should limit it to 1. Otherwise there are other interesting cases: 
![image](https://github.com/user-attachments/assets/b2ce5241-0950-4527-8b77-6a7c25f9aadf)